client/packages/common/src/hooks/useUrlQuery/useUrlQuery.ts

```diff
@@ -89,11 +85,14 @@ export const useUrlQuery = ({ skipParse = [] }: useUrlQueryProps = {}) => {
     setSearchParams(newQueryObject, { replace: true });
   };
 
-  return {
-    urlQuery: parseSearchParams(searchParams, skipParse),
-    updateQuery,
-    parseRangeString,
-  };
+  return useMemo(
+    () => ({
+      urlQuery: parseSearchParams(searchParams, skipParse ?? []),
+      updateQuery,
+      parseRangeString,
+    }),
+    [searchParams, skipParse]
```

Comment:
This useMemo was needed to avoid a render loop in the DetailTabs (useEffect that depends on the urlQuery)...